src/browser/components/Visualization/components/Map/index.ts

```diff
@@ -0,0 +1 @@
+export { Map } from "./Map";
```

Comment:
First visualization we are supposed to support is Chart so replace Map component with Chart.